front/src/BasicTable.tsx

```diff
@@ -9,42 +9,56 @@ import { useTranslation } from 'react-i18next';
 import { Button } from '@material-ui/core';
 import "./BasicTable.css";
 
-export default function BasicTable() {
-  const [offers, setOffers] = useState<any[]>([]); // TODO fix any
+
+interface OffersTypes {
+      _id: string,
+      position: string,
+      salary: string,
+      location: string,
+      company: string,
+      source: string,
```

Comment:
We support finite number of sources so you could type it more strict, consider `source: "bulldogjob" | "otherSite"...` Not sure, which exactly are supported so you need to determine this from e.g. api calls.